### ISSUE TITLE
Test failure on 32bit machines

### ISSUE THREAD

**Issue by kloetzl** [marbl/Mash#109]
If one sets `-DARCH_32` during compilation the tests fail as a different hash function is used compared to the reference. I propose to recreate 32bit tests or simply deactivate them on 32bit machines.

**Comment by ondovb** [comment]
Good catch, having two reference files for each test (32 and 64) would be ideal.
